README.md

```diff
@@ -73,11 +73,19 @@ Download `xtensor` as test package:
 ./download-test-package.sh
 ```
 
-Run test upload using quetz-client. (For testing purposes, an API key is created for the test user "alice" at server launch and is printed to the terminal, so use that for this example):
+To upload the package, install the [quetz-client](https://github.com/mamba-org/quetz-client):
+
+```bash
+mamba install quetz-client
+```
+
+To run the upload, you need to set environment variables for the quetz API key (which authenticates you) and the quetz server URL. As we passed the `--dev` flag, a testing API key can be found in quetz's output which you can use for this example.
 
 ```bash
 export QUETZ_API_KEY=E_KaBFstCKI9hTdPM7DQq56GglRHf2HW7tQtq6si370
-quetz-client http://localhost:8000/api/channels/channel0 xtensor/linux-64/xtensor-0.16.1-0.tar.bz2 xtensor/osx-64/xtensor-0.16.1-0.tar.bz2
+export QUETZ_SERVER_URL=http://localhost:8000
+quetz-client post_file_to_channel channel0 xtensor/linux-64/xtensor-0.16.1-0.tar.bz2 
```

Comment:
Is this the syntax of the quetz-client? I wonder if `upload` wouldn't be a nicer CLI command/

Reply:
We tried to stay as close as possible to the names of the routes in the Quetz server. But I wouldn't oppose a rename.

Reply:
I've opened a separate ticket for this: https://github.com/mamba-org/quetz-client/issues/14